packages/graphback-cli/src/helpers/initHelper.ts

```diff
@@ -79,6 +80,18 @@ async function assignTemplate(templateName: string): Promise<Template> {
   return template
 }
 
+function createDBConfig(name: string) {
+  const configPath = `${homedir}/.db.json`
```

Comment:
-10000 on db specific config. We will end up with 30 different config files in user root directory?

-10000 on actually having this globally. This could lead to really nasty bugs. 
It should be always project based not global